components/discord/actions/send-message/send-message.mjs

```diff
@@ -0,0 +1,36 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "discord-send-message",
+  name: "Send Message",
+  description: "Send a simple message to a Discord channel",
+  version: "0.0.1",
+  type: "action",
+  async run({ $ }) {
+    const {
+      message,
+      avatarURL,
+      threadID,
+      username,
+      includeSentViaPipedream,
+    } = this;
+
+    try {
+      const resp = await this.discord.createMessage(this.channel, {
```

Comment:
I didn't see the implementation of this function anywhere maybe I'm missing something here?

Reply:
The `createMessage` method is a special method built into the `discord` prop. See the [issue](https://github.com/PipedreamHQ/pipedream/issues/3419) for this action.